### PR TITLE
mix help when theres a task and alias

### PR DESCRIPTION
**Pull request by davydog187**
Addresses #8015

### CODE REVIEW

File: lib/mix/lib/mix/tasks/help.ex [hunk 1]
```diff
@@ -86,11 +86,40 @@ defmodule Mix.Tasks.Help do
...
+    case verbose_doc(task) do
+      [single] ->
+        print_doc(task, single, opts)
+
+      [alias_doc, task_doc] ->
+        print_doc(task, alias_doc, opts)
+
+        note = "\nThere is also a task named \"#{task}\". The documentation is shown next.\n"
+
+        if ansi_docs?(opts) do
```
Comment (fertapric) [inline lib/mix/lib/mix/tasks/help.ex]: I think this can be a simple `IO.puts` since there are not any special formatting.

Comment (davydog187) [reply lib/mix/lib/mix/tasks/help.ex]: What about width? I think it needs to be considered

Comment (fertapric) [reply lib/mix/lib/mix/tasks/help.ex]: You are right 😓 Sorry for the noise

File: lib/mix/lib/mix/tasks/help.ex [hunk 2]
```diff
@@ -86,11 +86,40 @@ defmodule Mix.Tasks.Help do
...
+      end
+
+    case verbose_doc(task) do
+      [single] ->
+        print_doc(task, single, opts)
+
+      [alias_doc, task_doc] ->
+        print_doc(task, alias_doc, opts)
+
+        note = "\nThere is also a task named \"#{task}\". The documentation is shown next.\n"
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/help.ex]: What if you add this note to `alias_doc` in the line above?

    alias_doc <> "\nThere is also a task named \"#{task}\". The documentation is shown next.\n"

This way you don't need to check for `ansi_docs?(opts)` and the like.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**